### PR TITLE
Release 0.1.4 admission-controller charts

### PR DESCRIPTION
**Pull request by michael-zhou-lw**

### CODE REVIEW

File: admission-controller/values.schema.json [hunk 1]
```diff
@@ -333,6 +335,15 @@
             ]
           }
         },
+        "blockOnError": {
+          "$id": "#/properties/scanner/properties/skipVerify",
```
Comment (ChitraPKumar) [inline admission-controller/values.schema.json:339]: Why does this say skipVerify?

File: admission-controller/values.yaml [hunk 2]
```diff
@@ -70,9 +71,9 @@ deployment:
 
 proxy-scanner:
   fullnameOverride: lacework-proxy-scanner
-  default_registry: index.docker.io
   enabled: true
   config:
+    default_registry: index.docker.io
```
Comment (ChitraPKumar) [inline admission-controller/values.yaml:76]: Has the corresponding parsing changed in AC code?

**Review by ChitraPKumar — COMMENTED** [review]
Have some basic questions. Can approve if that's expected. Thank you.

File: admission-controller/values.yaml (hunk 2 above)
Comment (michael-zhou-lw) [reply admission-controller/values.yaml:76]: No, realized that this parameter was in the wrong place before, proxy-scanner wasn't able to read it when it was outside `config`.

File: admission-controller/values.schema.json (hunk 1 above)
Comment (michael-zhou-lw) [reply admission-controller/values.schema.json]: Great catch, fixed.

**Review by ChitraPKumar — APPROVED** [review]

**Pull request merged**